contracts/IndirectEventEmitter.sol

```diff
@@ -2,8 +2,18 @@ pragma solidity ^0.5.0;
 
 contract IndirectEventEmitter {
     event IndirectString(string value);
+    event IndexedConflictingUint(uint256 indexed normalValue, uint256 indexed indexedConflictValue);
```

Comment:
Following up on the discussion about argument names. I think I'm just confused because here `normalValue` is actually indexed, and elsewhere `normalValue` is used for non-indexed arguments.

Can you rename this `nomalValue` to `indexedValue` or something else?

Reply:
Ok, I changed from

EventEmitter:
indexedValue -> indexedValue1
normalValue -> normalValue1

IndirectEventEmitter:
normalValue -> indexedValue2
indexedConflictValue-> indexedValue3

I agree that reusing name "normalValue" was confusing. I didn't wanted to repeat the indexedValue from the other contract, because I want to test specifically requesting the wrong event (with the wrong arg names).